### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by pat-s**
See commits.

### CODE REVIEW

File: release-config.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
 export default {
+  commentOnReleasedPullRequests: false,
+  releasePrefix: "Changelog for version",
```
Comment (anbraten) [inline release-config.ts:3]:
```suggestion
  commentOnReleasedPullRequests: false,
```

Wouldn't add that prefix as it is kinda misleading (more than just adjusting the changelog) and would also result in an commit with that text.

Comment (pat-s) [reply release-config.ts:3]: Yeah, I assumed you would comment on this/not like it. I see it's a subjective one. The same reasoning could also be brought up for the current default of "Release XY". 

I think the differences are between the different states (PR and commit message):

- in the PR state, I find the title "Release XY" misleading, as it suggests the release is happening very soon (even though it's usually a pending PR until some time). While in fact it actually just curates/updates the upcoming changelog in this state.
- After merge, the PR title matches well as it describes the "action" well in the commit log - and the plugin might in fact do the release/create tags, etc.

This is now partly off-topic, but what about this: 
- Have the PR title be something like "Changelog for XY" 
- Change the PR title during the merge to "Release XY" 

(I'll change this back now to not block this but maybe this one can also be outsourced into a real discussion into the plugin repo?)

**Comment by pat-s** [comment]
@anbraten Good to go now?

**Comment by pat-s** [comment]
ping @anbraten

**Review by anbraten — APPROVED** [review]

**Pull request merged**